service/app/controllers/usermanagement/UserController.java

```diff
@@ -39,6 +39,26 @@
 		  request.getContext().put(JsonKey.VERSION, JsonKey.VERSION_2); return null; },
 		  null, null, true, httpRequest);
 		 }
+  
+  
+  public CompletionStage<Result> createUserV3Sync(Http.Request httpRequest) {
```

Comment:
@Hari-stackroute can you share the response of the API.

Reply:
`{
    "id": "api.user.create",
    "ver": "v3",
    "ts": "2020-02-06 13:47:21:197+0530",
    "params": {
        "resmsgid": null,
        "msgid": null,
        "err": null,
        "status": "success",
        "errmsg": null
    },
    "responseCode": "OK",
    "result": {
        "userId": "6b534c6c-5a9a-441a-9d5e-9e2f98ab3616",
        "response": "SUCCESS",
        "errors": []
    }
}`